examples/imsim-skycat.yaml

```diff
@@ -0,0 +1,47 @@
+# Use baseline configuration
+template: /global/homes/j/jchiang8/dev/imSim/config/imsim-config.yaml
+
+# Disable inputs from instance catalogs.
+input.instance_catalog: ""
+
+# Use skyCatalogs for obtaining the objects to render.
+input.sky_catalog:
+    file_name: /global/homes/j/jchiang8/dev/imSim/tests/data/sky_cat_9683.yaml
+    obj_types: [galaxy]   # restrict to galaxies to avoid bright stars
+
+input.opsim_meta_dict.file_name: /global/homes/j/jchiang8/dev/imSim/tests/data/small_opsim_9683.db
+input.opsim_meta_dict.visit: 449053
+
+input.tree_rings.only_dets: [R22_S11]
+
+# Disable the atmospheric PSF to run faster for testing.
+input.atm_psf: ""
+
+image.nobjects: 1000
+image.random_seed: 42
+
+image.wcs.boresight:
+    type: RADec
+    ra:
+        type: Degrees
+        theta: { type: OpsimMeta, field: fieldRA }
+    dec:
+        type: Degrees
+        theta: { type: OpsimMeta, field: fieldDec }
```

Comment:
I still think this should be swapped in the original rather than replaced here.  I.e. Have `image.wcs.boresight` in imsim-config.yaml use this specification.  And then have `input.atm_psf.boresight` be `'@image.wcs.boresight'`.  Then, this override in the user config could be removed.